modules/openvino_code/README.md

```diff
@@ -35,41 +19,5 @@ TDB
 
 You can see input to and output from the code generation API:
 
-1. Open VSCode `OUTPUT` panel
-2. Select extension output source from the dropdown menu
-
-## Developing
```

Comment:
I would suggest not to just remove development related sections from README, but to move them to separate file (e.g. `DEVELOPMENT.md`) as described in corresponding ticket.

Reply:
Yes, we will create `DEVELOPMENT.md` in a separate PR.